clipper_admin/clipper_admin/kubernetes/prom_depolyment.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    ai.clipper.container.label: ""
+    ai.clipper.name: metrics
+  name: metrics
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        ai.clipper.container.label: ""
+        ai.clipper.name: metrics
+    spec:
+      containers:
+      - image: prom/prometheus:latest
+        args: 
+        - "--config.file=/etc/prometheus/prometheus.yml"
+        - "--storage.tsdb.path=/prometheus"
+        - "--web.console.libraries=/etc/prometheus/console_libraries"
+        - "--web.console.templates=/etc/prometheus/consoles"
+        - "--web.enable-lifecycle"
+        name: metrics
+        ports:
+        - containerPort: 1390
+        volumeMounts:
+        - name: config-volume #This name needs to match the one under "volumes"
+          mountPath: /etc/prometheus
+      volumes:
```

Comment:
What is the volume for?

Reply:
This is a volume to configMap, it overrides the default prometheus configuration